figures/Comparison-ERP-TRF.py

```diff
@@ -0,0 +1,180 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: tutorialAlice
+#     language: python
+#     name: tutorialalice
+# ---
+
+import eelbrain
+import os
+from pathlib import Path
+import copy
+import numpy as np
+from matplotlib import pyplot
+
+# +
+# Data locations
+DATA_ROOT = Path("~").expanduser() / 'Data' / 'Alice'
+PREDICTOR_DIR = DATA_ROOT / 'predictors'
+STIMULI_DIR = DATA_ROOT / 'stimuli'
+TRF_DIR = DATA_ROOT / 'TRFs'
+EPOCH_DIR = DATA_ROOT / 'Epochs'
+
+tstart = -0.1
+tstop = 1
+# -
+
+# Where to save the figure
+DST = DATA_ROOT / 'figures'
+DST.mkdir(exist_ok=True)
+
+# get all subjects
+subjects = [subject for subject in os.listdir(TRF_DIR) if subject.startswith('S') ]
+
+# (1) GET THE ERP RESPONSE TO A WORD ONSET
```

Comment:
If epoching is not simple enough to do on the fly here, there should probably be a comment somewhere warns the user that the make epochs script has to be run first?